### PR TITLE
Added gallery section

### PR DESCRIPTION
**Pull request by ReinisZogots**

### CODE REVIEW

File: src/components/gallery/index.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import Structure from "../../structure.json";
+import "./Gallery.scss";
+
+const Gallery = () => {
+  const Gallery = Structure.gallery;
+  const Images = Gallery.galleryImages;
+  const rearrangedImages = [
```
Comment (Jalal999) [inline src/components/gallery/index.jsx:8]: what if new image is added to json? will it be shown in this section? 
this way is kind of hardcode

Comment (ReinisZogots) [reply src/components/gallery/index.jsx:8]: In the given design the images are shown in a different order than it is given in JSON file. Is there a way besides hardcoding it to arrange them without changing the JSON file?

File: src/components/gallery/index.jsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    Images[3],
+  ];
+  return (
+    <div className="gallery">
+      <div className="gallery__text-container">
+        <h2 className="gallery__heading">{Gallery.title}</h2>
+        <h4 className="gallery__subheading">{Gallery.subTitle}</h4>
+      </div>
+      <div className="gallery__image-container">
+        {rearrangedImages.map((image, index) => {
```
Comment (VSlobolinskyi) [inline src/components/gallery/index.jsx:23]: Why are you rearranging images in the first place?

Comment (ReinisZogots) [reply src/components/gallery/index.jsx:23]: the order in the design is different from JSON file

**Review by VSlobolinskyi — APPROVED** [review]

**Review by Jalal999 — APPROVED** [review]

File: src/components/gallery/index.jsx (hunk 1 above)
Comment (YuraLeo94) [reply src/components/gallery/index.jsx:8]: In this case, do changes in your data in JSON and avoid hardcode

File: src/components/gallery/index.jsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+          const { path, altTag } = image;
+          return (
+            <img
+              className="gallery__image"
+              src={`${process.env.PUBLIC_URL} ${path}`}
```
Comment (YuraLeo94) [inline src/components/gallery/index.jsx:28]: `${image.path}`

File: src/components/gallery/index.jsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+              alt={altTag}
```
Comment (YuraLeo94) [inline src/components/gallery/index.jsx:29]: `image.altTag`

**Review by YuraLeo94 — CHANGES_REQUESTED** [review]

File: src/components/gallery/index.jsx [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+  ];
+  return (
+    <div className="gallery">
+      <div className="gallery__text-container">
+        <h2 className="gallery__heading">{Gallery.title}</h2>
+        <h4 className="gallery__subheading">{Gallery.subTitle}</h4>
+      </div>
+      <div className="gallery__image-container">
+        {rearrangedImages.map((image, index) => {
+          const { path, altTag } = image;
```
[the remark this answers is not in the document]
Comment (ReinisZogots) [reply src/components/gallery/index.jsx]: removed

File: src/components/gallery/index.jsx (hunk 3 above)
Comment (ReinisZogots) [reply src/components/gallery/index.jsx]: added

File: src/components/gallery/index.jsx (hunk 4 above)
Comment (ReinisZogots) [reply src/components/gallery/index.jsx]: added

File: src/components/gallery/index.jsx (hunk 1 above)
Comment (ReinisZogots) [reply src/components/gallery/index.jsx]: Edited JSON file

**Review by YuraLeo94 — APPROVED** [review]

**Pull request merged**